portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,16 @@ function getRandomFactAboutRoss() {
 
   // Pick a random fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
-
+  let currentFactIndex = facts.indexOf(fact);
+  if(currentFactIndex === 2 || currentFactIndex === 3) {
+    document.getElementById("map").style.width = "400px"; 
+    document.getElementById("map").style.height = "400px";
+    showMap();
+  }
+  else{
+    document.getElementById("map").style.width = "0px"; 
+    document.getElementById("map").style.height = "0px";
```

Comment:
create a variable outside the if / else statement, then change it's style inside the if / else statement